Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Products.cs

```diff
@@ -12,15 +12,17 @@ public Products()
             Initialise();
         }
         public int ProductsId { get; set; }
-        public string Name { get; set; }
-        public string CoverImageLocation { get; set; }
-        public string Description { get; set; }
-        public int Rating { get; set; }
-        public double Price { get; set; }
-        public string Type { get; set; }
-        public int QuantityOnHand { get; set; }
+        public string ProductName { get; set; }
+        public string ProductCoverImageLocation { get; set; }
+        public string ProductDescription { get; set; }
+        public int? ProductRating { get; set; }
+        public double ProductPrice { get; set; }
+        public string ProductType { get; set; }
+        public int ProductQuantityOnHand { get; set; }
+        public int? ProductSizeOption { get; set; }
+        public int? ProductColourOption { get; set; }
         public ICollection<ProductOrders> ProductOrder { get; set; }
```

Comment:
Also a small thing but since you've asked for it, there should be a space between line 24 and 25 but that does not stop u from compiling

Reply:
I had these lines initially then removed them all for whitespaces zzz